src/sql/azdata.proposed.d.ts

```diff
@@ -2612,6 +2612,8 @@ declare module 'azdata' {
 	export class TreeItem extends vscode.TreeItem {
 		payload?: IConnectionProfile;
 		childProvider?: string;
+		isServer?: boolean;
+		isDatabase?: boolean;
```

Comment:
Do you ever see there ever being a case where isServer/isDatabase are both true or false? I'd just combine these and use a single var, isDatabase seems enough (where isDatabase would be false for server nodes)

Reply:
You never even seem to use isServer and/or isDatabase directly. Why not just have isConnectable?

---
In reply to: [297377424](https://github.com/microsoft/azuredatastudio/pull/6158#discussion_r297377424) [](ancestors = 297377424)